CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1743,7 +1752,7 @@ private void OnLoaded(object sender, RoutedEventArgs routedEventArgs)
             tooltipTimer.IsEnabled = false;
 
             //Initial value for screen location
-            browserScreenLocation = PointToScreen(new Point());
+            updateBrowserScreenLocation();
```

Comment:
`browserScreenLocation` would need a default value here. Perhaps make a call to `updateBrowserScreenLocation` once the control is part of a `visual tree` (assuming there is an event for that)